app/shared/src/desktopTest/kotlin/ui/progress/MediaProgressSliderStateTest.kt

```diff
@@ -0,0 +1,87 @@
+package ui.progress
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableLongStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import me.him188.ani.app.ui.framework.runComposeStateTest
+import me.him188.ani.app.videoplayer.ui.progress.MediaProgressSliderState
+import me.him188.ani.app.videoplayer.ui.state.Chapter
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import kotlin.time.Duration.Companion.seconds
+
+class MediaProgressSliderStateTest {
+
+    @Test
+    fun testMediaProgressSliderState() = runComposeStateTest {
+        val chapters = object : ImmutableList<Chapter> {
+            override val size: Int
+                get() = TODO("Not yet implemented")
+
+            override fun get(index: Int): Chapter {
+                TODO("Not yet implemented")
+            }
+
+            override fun isEmpty(): Boolean {
+                TODO("Not yet implemented")
+            }
+
+            override fun iterator(): Iterator<Chapter> {
+                TODO("Not yet implemented")
+            }
+
+            override fun listIterator(): ListIterator<Chapter> {
+                TODO("Not yet implemented")
+            }
+
+            override fun listIterator(index: Int): ListIterator<Chapter> {
+                TODO("Not yet implemented")
+            }
+
+            override fun lastIndexOf(element: Chapter): Int {
+                TODO("Not yet implemented")
+            }
+
+            override fun indexOf(element: Chapter): Int {
+                TODO("Not yet implemented")
+            }
+
+            override fun containsAll(elements: Collection<Chapter>): Boolean {
+                TODO("Not yet implemented")
+            }
+
+            override fun contains(element: Chapter): Boolean {
+                TODO("Not yet implemented")
+            }
+
+        }
+        var positionState by mutableLongStateOf(0L)
+        val scope = CoroutineScope(Dispatchers.Main)
+        scope.launch {
```

Comment:
```suggestion
        launch {
```

test 是单线程的, 主线程 Main 和当前协程是一个单线程调度器 `TestDispatcher` 跑的